pkg/resources/helpers_test.go

```diff
@@ -89,6 +89,14 @@ func table(t *testing.T, id string, params map[string]interface{}) *schema.Resou
 	return d
 }
 
+func stream(t *testing.T, id string, params map[string]interface{}) *schema.ResourceData {
```

Comment:
Put this in alphabetical order as well, so on line 84.

Reply:
done